opensea-operator-filter/README.md

```diff
@@ -0,0 +1,248 @@
+# ERC721WithOperatorFitlerer
+
+## Overview
+
+Starting January 2nd, 2023, Opensea will begin validating creator fee enforcement for new collections on all supported EVM chains. After January 2nd, 2023, if OpenSea is unable to validate enforcement, OpenSea will make creator fees optional for that collection. Older collections will continue to have their fees enforced on OpenSea, including on Ethereum Mainnet (previously, enforcement was already required on Ethereum Mainnet).
+
+This module provides ERC721 contract with operator filterer functions.
+
+Token contracts that wish to manage lists of filtered operators and restrict transfers from them may integrate with the registry easily with tokens using the `OperatorFilterer` and `DefaultOperatorFilterer` contracts. These contracts provide modifiers (`onlyAllowedOperator` and `onlyAllowedOperatorApproval`) which can be used on the token's transfer methods to restrict transfers from or approvals of filtered operators.
+
+The module provides the functions to create a nft collection and host its metadata on their own using a third party service like s3.
```

Comment:
Of course, we use S3 bucket as uploading metadata host service, but it should be decentralized so it can be used in NFT.
Adding the description how to pin centralized link to decentralized node would be better.